src/lib/services/template-service.ts

```
@@ -0,0 +1,498 @@
+export class TemplateService {
+	private static templatesDir = path.join(process.cwd(), 'data/templates');
+	private static composeTemplatesDir = path.join(this.templatesDir, 'compose');
+	private static envTemplateFile = path.join(this.templatesDir, '.env.template');
+
+	/**
+	 * Get all available compose templates from the file system
+	 */
+	static async getComposeTemplates(): Promise<ComposeTemplate[]> {
+		try {
+			// Ensure directories exist
+			await this.ensureDirectoriesExist();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)